### ISSUE TITLE
Rename --uri to --concept or add alias

### ISSUE THREAD

**Issue by nichtich** [scriptotek/mc2skos#60]
Concept URI template is specified with `concept` in `vocabularies.yaml` and with `--uri` on command line. The latter should be renamed to `--concept` to match names.

**Comment by danmichaelo** [comment]
Yeah, agree. We can add `--concept` and deprecate `--uri`. Feel free to add a PR if you have time.
